README.md

```diff
@@ -8,24 +8,90 @@ ospool:
+
+	The job exceeded allowed execute duration of 20:00:00
+
+<li> You have reached your disk quota and HTCondor can not write the output file to your access point. This may happen while performing [checkpointing](https://portal.osg-htc.org/documentation/htc_workloads/submitting_workloads/checkpointing-on-OSPool/) jobs or regular jobs when your disk quota is reached. The hold message will look like:</li>
+
+	Transfer output files failure at access point… while receiving files from the execution point. Details: Error from ….execute point … failed to send file(s) at apxx; failed to create directory /path Disk quota exceeded
+</ul>
+For these cases, sometimes it is better to just remove the jobs, fix the issues and then resubmit them. In some cases (e.g. excessive memory usage) using [condor_qedit](https://htcondor.readthedocs.io/en/latest/man-pages/condor_qedit.html) to fix the issue and releasing them might be convenient. Whenever in doubt please reach out to our [support](https://portal.osg-htc.org/documentation/support_and_training/support/getting-help-from-RCFs/) team.  
+
+## Job completed but was unsuccessful
+Under this case two things will be considered. First, your job has run but the code did not execute correctly or in the expected manner. Second, your job ran but it did not produce/transfer the desired output files.
+
+For these scenarios looking at the `error` and `log` files will provide more information about your job. In addition, having various `ls` or `echo` statements that shows the contents of the working directory and the code's progression are helpful in diagnosing the actual issue. An example is shown here:
+
+	echo "Here are the files at the start of the job:"
+	ls -R
+	echo "Executing main command..."
+	my_command	#Part of the original code
+	echo "... finished. Here are the files at the end of the job:"
+	ls -R
+
+While using the above commands, looking at the `output` file will provide the status of your code. In addition, for your commands if enabling debugging or verbose logging is an option then do those. Moreover, [condor_chirp](https://www.google.com/url?q=https://htcondor.readthedocs.io/en/latest/man-pages/condor_chirp.html&sa=D&source=editors&ust=1740162417565289&usg=AOvVaw1HJ8p454psoN6alPpujatn) can be a very useful tool in this regard as it sends information directly back to the Access Point. For example, the following will add a statement to your .log file:
+	command1
+	condor_chirp ulog "Finished executing command1"
+	command2
+## Job does not start
+
+Matchmaking cycle (the process of finding an execution point to run the submitted jobs in queue) can take more than 5 minutes to complete and it can be longer if the server is busy. In addition, the more/more-specific resources are requested, typically the longer is the wait. For example if you are submitting a lot of GPU jobs, there may be fewer `GPUs` (than other resources) to run your jobs. This type of issue can be diagnosed using the `-better-analyze` flag 
```

Comment:
```suggestion
The matchmaking cycle (the process of finding an execution point to run the submitted jobs in queue) can take more than 5 minutes to complete and it can be longer if the server is busy. In addition, the more/more-specific resources are requested, typically the longer is the wait. For example if you are submitting a lot of GPU jobs, there may be fewer `GPUs` (than other resources) to run your jobs. This type of issue can be diagnosed using the `-better-analyze` flag 
```